DiamondLightSource/python-artemis
# I23 grid scans

Fixes #578 and fixes #574

Link to dodal PR (if required): #59 (https://github.com/DiamondLightSource/dodal/pull/59)

This change introduces the stepped_grid_scan end point and the associated changes required to perform the basic movement required for a stepped grid scan.

### To test:
1. Ensure https://gerrit.diamond.ac.uk/c/gda/gda-mx/+/38922 is installed into GDA.
2. Run GDA for i23
3. Press the "Xray Centring" button on the OAV controls area
4. Within Hyperion confirm that the stepped grid scan plan is called and the x & y motors are moved
